### PR TITLE
handle bundle upgrades

### PR DESCRIPTION
**Pull request by tchughesiv**
```bash
$ make generate bundle bundle-build bundle-push catalog-build catalog-push \
    IMG=quay.io/mongodb/mongodb-atlas-kubernetes-dbaas \
    CATALOG_BASE_IMG=quay.io/mongodb/mongodb-atlas-kubernetes-dbaas-catalog:0.2.0 \
    VERSION=0.3.0
```

Signed-off-by: Tommy Hughes <tohughes@redhat.com>

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -6,7 +6,9 @@ SHELL := /usr/bin/env bash
 # To re-generate a bundle for another specific version without changing the standard setup, you can:
 # - use the VERSION as arg of the bundle target (e.g make bundle VERSION=0.0.2)
 # - use environment variables to overwrite this value (e.g export VERSION=0.0.2)
-VERSION ?= 0.5.0
+VERSION ?= 0.3.0
```
Comment (jianrongzhang89) [inline Makefile:9]: should this be 0.8.0 instead?

Comment (tchughesiv) [reply Makefile:9]: why 0.8.0?

Comment (jianrongzhang89) [reply Makefile:9]: This is the version used in the upstream currently: https://github.com/mongodb/mongodb-atlas-kubernetes/blob/main/Makefile

Comment (tchughesiv) [reply Makefile:9]: Right, but here’s what we use in RHODA.
https://github.com/RHEcosystemAppEng/dbaas-operator/blob/release-0.1.5/controllers/reconcilers/constants.go

I’m good with changing though. Just have to pass right version during make

Comment (jianrongzhang89) [reply Makefile:9]: Yes, needs to pass in the right version during the build, just as what was done in previous build of RHODA. Changing it to 0.8.0 can make rebasing easier and less confusing.

Comment (tchughesiv) [reply Makefile]: done

**Comment by jianrongzhang89** [comment]
LGTM

**Pull request merged**